src/token/populate-user.js

```diff
@@ -0,0 +1,35 @@
+import Debug from 'debug';
+
+const debug = Debug('feathers-authentication:token:populate-user');
+
+export default function(options) {
+  const app = this;
+  const { user } = options;
+
+  if(!user.service) {
+    throw new Error(`'user.service' needs to be set in authentication options`);
+  }
+
+  return function populateUser(data) {
+    const service = app.service(user.service);
```

Comment:
Possibly we want to support passing an actual service as well, instead of just the path name.